docs/cuda-miner.md

```diff
@@ -131,7 +129,7 @@ mining:
 After updating the configuration, the node should be started (or restarted if it's already running):
 
 ```
-~/aeternity/node/bin/epoch start
+~/aeternity/node/bin/aeternity start ## Before version 1.3.0, the executable used to be called `epoch` rather than `aeternity`.
```

Comment:
Why the clarification, the docs are versioned as well ?

Reply:
My problem was that this section claims to have been tested:
> The documentation in this section is tested on:
> - Epoch version 1.0.0-rc2

Hence I am trying not to misrepresent that.